kyaml/kio/filters/testdata/dataset1-expected/java/java-deployment.resource.yaml

```diff
@@ -37,7 +37,6 @@ spec:
             name: app-config
         env:
         - name: JAVA_OPTS
-          value: -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
-            -Djava.security.egd=file:/dev/./urandom
```

Comment:
It looks like the input had this on two lines, so this seems like a regression to me. But not related to Mike's PR.